public/app/features/datasources/components/ButtonRow.tsx

```diff
@@ -31,6 +31,7 @@ export function ButtonRow({ canSave, canDelete, onDelete, onSubmit, onTest }: Pr
           disabled={!canSave}
           onClick={onSubmit}
           data-testid={selectors.pages.DataSource.saveAndTest}
+          id={selectors.pages.DataSource.saveAndTest}
```

Comment:
nit: what was the reason to add this id?

Reply:
@hugohaggmark - I added it so I can target the id on the left nav bar I added can scroll down to it on click.